tests/foreman/endtoend/test_cli_endtoend.py

```diff
@@ -110,7 +110,7 @@ def test_positive_cli_end_to_end(function_entitlement_manifest, target_sat, rhel
 
     # step 2.1: Create a new organization
     org = _create(user, target_sat.cli.Org, {'name': gen_alphanumeric()})
-    target_sat.cli.SimpleContentAccess.disable({'organization-id': org['id']})
+    # target_sat.cli.SimpleContentAccess.disable({'organization-id': org['id']})
```

Comment:
This can be removed for good in master/6.16since it is SCA-only.

Also I would recommend to use `function_sca_manifest` instead and remove [this](https://github.com/SatelliteQE/robottelo/pull/13999/files#diff-bd4df4de4718dbede609d2c4ef760267922eed1f32cdd08006abd3e4fee09b39R232-R242) as not applicable in the SCA-only org.